mlx-mkbfb

```diff
@@ -93,6 +93,7 @@ image_list = (
     ("boot-path", "Boot image path", 58),
     ("boot-args", "Arguments for boot image", 59),
     ("boot-timeout", "Boot menu timeout", 60),
+    ("nicfw", "NIC Firmware image", 49),
```

Comment:
Question: Can we re-use the ID of the "dummy" line 98? Or do we still need the "dummy"?

Reply:
Done. I don't see dummy is used anywhere, thus replacing it with 'nicfw'.

Reply:
Thanks!